students/ZhuojingXie/lesson03/list_lab.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
+
+
+
+#series 1
+list1 =  ["Apples", "Pears", "Oranges" , "Peaches"]
+print(list1)
+fruit1 = input("Tell me a fruit you like > ")
+list1.append(fruit1)
+print(list1)
+num1 = int(input("Now, tell me a numer you like between 1 to {} > ".format(len(list1))))
+while num1<0 or num1 > len(list1):
```

Comment:
another way it can be represented is:
`while not 1 <= num <= len(list1)`

which is saying: while number is not between 1 and size of list inclusive keep looping

Reply:
you want to try and minimize duplicate or common code, so instead of:
```
num1 = int(input("Now, tell me a numer you like between 1 to {} > ".format(len(list1))))
while num1<0 or num1 > len(list1):
    num1 = int(input("Please pick up a number between 1 to {} ".format(len(list1))))
```
you can have a more concise version:
```
num1 = 0
while not 1 <= num1 <= len(list1):
    num1 = int(input(".... ")
```